### ISSUE TITLE
Add .buggerc support

### ISSUE THREAD

**Issue by hybrist** [buggerjs/bugger#28]
Maybe consider having some sort of defaults-file (home directory or in project) that can be used to shorten the command being run or alter the defaults. E.g. making `--watch` default or by default starting without a break point.

**Comment by caspervonb** [comment]
Wouldn't an alias be just as good? The amount of options is rather minimal?

**Comment by jkrems** [comment]
Yeah, this was created when it had many more option (multiple ports, support for require hooks for coffee-script entry scripts etc.).
